src/main/java/org/opentripplanner/api/resource/GraphPathToTripPlanConverter.java

```diff
@@ -163,7 +163,7 @@ public static Itinerary generateItinerary(GraphPath path, boolean showIntermedia
         fixupLegs(itinerary.legs, legsStates);
 
         itinerary.duration = lastState.getElapsedTimeSeconds();
-        itinerary.startTime = makeCalendar(states[0]);
+        itinerary.startTime = getStartTime(states);
         itinerary.endTime = makeCalendar(lastState);
```

Comment:
Is it possible for this to happen with the lastState?

Reply:
Very good point! Yes, it can. Fixed. 

**Before:**
<img width="1297" alt="image" src="https://user-images.githubusercontent.com/45011335/62728075-d9415700-b9e8-11e9-8cfe-3f2df0740641.png">

**After:**
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/45011335/62728113-ebbb9080-b9e8-11e9-90ea-a06fbd3c1e50.png">